layers/+lang/emacs-lisp/packages.el

```diff
@@ -160,6 +161,14 @@
   (use-package elisp-def
     :defer t))
 
+(defun emacs-lisp/init-elisp-demos ()
+  (use-package elisp-demos
+    :defer t)
+  (advice-add 'describe-function-1
+              :after #'elisp-demos-advice-describe-function-1)
+  (advice-add 'helpful-update
+              :after #'elisp-demos-advice-helpful-update))
```

Comment:
This should go into the use-package init or config block otherwise reading the file during loading will have unintended side effects.

Reply:
Done. Somehow this went unnoticed to me when reading the layer creation docs. forgive my newbieness. In addition, I'm starting to see some added benefits of moving some `user-config` code to private layers to make my config tidier.
Thanks for teaching me.